packages/framework/assets/js/admin/components/FileUpload.js

```diff
@@ -1,10 +1,10 @@
 import './jquery.dmuploader';
-import FileItem from './fileUpload.fileItem';
-import Window from '../utils/window';
-import { forceValidateElement } from '../validation/customization/validation';
-import formChangeInfo from '../components/formChangeInfo';
-import Ajax from '../../common/utils/ajax';
-import Register from '../../common/utils/register';
+import FileItem from './FileItem';
+import Window from '../utils/Window';
+import { forceValidateElement } from '../validation/customization/validationInit';
```

Comment:
Same question as the one above.

Reply:
374c67a886d912b7eaaaeec57089a93414dca4d8